tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_Translation.cpp

```diff
@@ -150,7 +201,15 @@ void test_translation() {
                   f_of_t_list);
     test_inv_jacobian(radial_translation_map_deserialized, point_xi, t,
                       f_of_t_list);
-
+    // This is for an edge case where the numerical derivative taken in the
+    // TestMapHelpers jacobian pushes the coordinate to the next boundary
+    // causing it to be translated improperly.
+    if (inspiral_radius <= 0.9997) {
```

Comment:
Could you explain a bit more what causes this? (github comment) I'm not sure I understand.

Reply:
Oh yeah, so in the numerical derivative function used in the jacobian, it calls the operator with tiny steps forward something like x1 = x + dx, x2 = x1 + dx and x3 = x2 + dx where dx = 1e-4 so it moves the point_xi up to 3e-4 forward and calls the operator with each different value. When it does that, it enters the region where the translation is different causing the jacobian to not be the identity matrix anymore. However, when it calls the jacobian, it still uses the original point_xi so there's a mixup between which region it's in. So to stop this from causing problems I just made it so that the test doesn't get called if your radius > 0.9997. I think this fix is fine because this problem can only arise when comparing the analytic jacobian to the numerical one, so this shouldn't affect evolutions or be thought of as a bug I don't think. However, this did remind me that I meant to put `or inspiral_radius > 1.0`

Reply:
Ahh ok that makes sense, thank you! I'd suggest a couple things:

- Instead of hard-coding `0.9997`, do `inspiral_radius <= 0.99 * inner_radius`. This gives you a safe-enough buffer and also doesn't hard code a value.
- I think it can also be `or inspiral_radius >= inner_radius`
- Can you add basically this exact github comment into the code? It was a very helpful explanation and I think the test readability would benefit from it.